src/report_1.R

```diff
@@ -1,3 +1,169 @@
-
+library(tidyverse)
+library(lubridate)
+library(tidyquant)
+library(extrafont)
+library(beepr)
+library(ggthemes)
+source("src/process_counts.R")
+# loadfonts() # <- possibly not needed
 ### `font_import()` should be run initially to ensure required fonts are present
 
+month_start <- function(month) {
+  as.Date(paste0(month, "-01"))
+}
+
+month_end <- function(month) {
+  as.Date(paste0(month, "-01")) + months(1) - days(1)
+}
+
+date_between <- function(start, end) {
+  out <- numeric(length = length(start))
+  for(i in seq_along(start)) {
+    out[i] <- sample(seq(min(start[i], end[i]),
+                         max(start[i], end[i]),
+                         by = "day"), 1)
+  }
+  as.Date(out)
+}
+
+impute_birthday <- function(birth_month, min_start, max_cease) {
+  earliest_possible <- max(max_cease - days(floor(18 * 365.25)) + 1, month_start(birth_month))
+  latest_possible <- min(min_start, month_end(birth_month))
+  date_between(earliest_possible, latest_possible)
+}
+
+year_diff <- function(start, stop) {
+  as.numeric(difftime(stop, start, units = "days")) %/% 365.25
+}
+
+theme_mastodon <- theme(plot.title = element_text(
+  hjust = 0.5, size = 20,
+  margin = margin(0,0,15,0)),
+  plot.caption = element_text(hjust = 0.5),
+  axis.title = element_text(
+    hjust = 0.5, size = 16),
+  axis.text = element_text(
+    hjust = 0.5, size = 10),
+  axis.text.x = element_text(angle = -45),
+  axis.title.x = element_text(margin = margin(15,0,0,0)),
+  axis.title.y = element_text(margin = margin(0,10,0,0)),
+  plot.margin = margin(10,20,10,10),
+  panel.grid = element_line(color = "#eeeeee"),
+  panel.background = element_rect(fill = "white",
+                                  colour = "black"))
+
+project_from <- as.Date("2020-03-31")
```

Comment:
magic date?

Reply:
yes exactly, my aim is to pull this in from the model run config, perhaps I'll leave a comment here